### DISCUSSION TITLE
Pix-to-Pix as Viton?

### DISCUSSION THREAD

**Discussion by Bilal143260** [huggingface/diffusers#6624] [Q&A]
Is it a good idea to train pix-to-pix-instruct to act like a viton (virtual try-on)? The training data would include images of cloth and prompts as input and a model wearing that dress as ground truth.

**Comment by sayakpaul** [comment]
I think it's better to use IP Adapters. I responded the same thing in the issue you had created.

**Reply by Bilal143260** [reply]
Yeah. I am training it. Hoping to get some good results

**Comment by a-r-r-o-w** [comment]
Have you given existing VTON papers a try btw? I haven't played around much with training but I feel directly training Pix2Pix on cloth-model pairs may not work. A few current approaches involve learning separate embeddings for clothes and applying a warping module for cloth placement.

- [ladi-vton](https://github.com/miccunifi/ladi-vton)
- [DCI-vton](https://github.com/bcmi/DCI-VTON-Virtual-Try-On)
- [TryOnDiffusion](https://github.com/tryonlabs/tryondiffusion)

> I think it's better to use IP Adapters. I responded the same thing in the issue you had created.

They're mostly unable to handle cloth consistency from what I've seen, but it can do a decent job at times with the right prompting + img2img and correct parameters.

**Reply by Bilal143260** [reply]
Exisiting vitons lose the texture details. The best I tested is Stable viton
